application/blueprints/main/views.py

```
@@ -206,6 +205,32 @@ def check_for_redirects(page):
     return redirects[page] if page in redirects else page
 
 
+# At the top of application/blueprints/main/views.py
+import logging
+import csv
```

Ruff (0.12.2)

Redefinition of unused `csv` from line 1

Remove definition: `csv`

(F811)